lib/exercism/user_exercise.rb

```diff
@@ -11,7 +11,7 @@ class UserExercise < ActiveRecord::Base
       .where('views.user_id': user.id)
       .where('views.last_viewed_at > ?', 30.days.ago).order('views.last_viewed_at DESC')
   }
-  scope :current, -> { where(archived: false).where.not(iteration_count: 0).order('language, id ASC') }
+  scope :current, -> { where(archived: false).where.not(iteration_count: 0).order('last_iteration_at DESC') }
```

Comment:
I think just sorting by most recent makes sense since the dashboard view only shows up to five exercises. The [other views]( exercism.io/app/views/user/_exercises_table.erb ) that show all the current exercises get sorted on the client with javascript, so I don't think it matters how it's served.